### DISCUSSION TITLE
locale in KDE

### DISCUSSION THREAD

**Discussion by waternumbers** [SpiralLinux/SpiralLinux-project#132] [Q&A] [answered]
Hi

Liking the KDE version of SpiralLinux but...

Installing from the live media I select settings for the UK: en-gb language, keyboard layout and London time zone.
On installation only the time zone is correct, keyboard and language are set to en-us.
Also the en-gb language packs for firefox, hunspell and libreoffice are not installed (a wireless connection was available in the live session)

Given the language support link on desktop is this a bug or expected behaviour?

If it is expected behaviour it is easy to install the language packs and alter the system settings for one user - but what is the best way of altering the settings for multiple users?

Thanks

Paul

**Comment by geckolinux** [comment, marked as answer]
Hi Paul, thanks for reporting the issue.

It looks like I included a few mistaken config files in the default Plasma configuration. I'll try to get that fixed in the next release.

Fortunately it's easy to just change the language as follows:

![Screenshot from 2022-09-28 17-52-52](https://user-images.githubusercontent.com/2692138/192903558-626aa068-91f0-48f2-ba3c-457a26ee8faf.png)

![Screenshot from 2022-09-28 18-02-52](https://user-images.githubusercontent.com/2692138/192903561-1d0c5d58-cca5-41e2-b88a-784b136756eb.png)

Or, you can delete the following config files for all users and then logout/login:
- `~/.config/kxkbrc` 
- `~/.config/plasma-localerc`
- `~/.config/KDE/Sonnet.conf`
- (You can also delete those same filenames from `/etc/skel/.config/` to prevent it from applying those files to newly created user accounts.)

> Also the en-gb language packs for firefox, hunspell and libreoffice are not installed

This is the expected behavior, Calamares doesn't download language packs. Please use the **Language support** utility to install additional language packs via Synaptic.

Thanks again for the report!

**Comment by waternumbers** [comment]
Thanks geckolinux, your answer above works. 

I also needed to remove `~/.config/KDE/Sonnet.conf` (or `/etc/skel/.config/KDE/Sonnet.conf` for new user accounts) to get en-GB spell checking set as the default

**Reply by geckolinux** [reply]
Excellent, thanks for confirming that.
